### PR TITLE
Added rubygem-hammer_cli_foreman_discovery 0.0.1

### PR DESCRIPTION
**Pull request by lzap**
EL6 scratch: http://koji.katello.org/koji/taskinfo?taskID=220918

### CODE REVIEW

File: rubygem-hammer_cli_foreman_discovery/rubygem-hammer_cli_foreman_discovery.spec [hunk 1]
```diff
@@ -0,0 +1,61 @@
+%global gem_name hammer_cli_foreman_discovery
+%global confdir hammer
+
+Summary: Foreman discovery commands for Hammer CLI
+Name: rubygem-%{gem_name}
+Version: 0.0.1
+Release: 1%{?dist}
+Group: Applications/System
+License: GPLv3
+URL: http://github.com/theforeman/hammer_cli_foreman_discovery
```
Comment (domcleal) [inline rubygem-hammer_cli_foreman_discovery/rubygem-hammer_cli_foreman_discovery.spec]: 404, hyphens

File: rubygem-hammer_cli_foreman_discovery/rubygem-hammer_cli_foreman_discovery.spec [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+%files
+%dir %{gem_instdir}
+%{gem_instdir}/lib
+%{gem_instdir}/locale
+%exclude %{gem_cache}
+%{gem_spec}
+
+%files doc
+%doc %{gem_dir}/doc/%{gem_name}-%{version}
```
Comment (domcleal) [inline rubygem-hammer_cli_foreman_discovery/rubygem-hammer_cli_foreman_discovery.spec]: %{gem_docdir}

**Comment by domcleal** [comment]
It seems to be missing a /etc/hammer/cli.modules.d/ file to activate the module.  It's best if this is included in the project repo and gem, then we can copy it in - but for 0.0.1 you may need to commit it into this repo temporarily too, unless you want to do 0.0.2.

**Comment by lzap** [comment]
Fixed all of these.

http://koji.katello.org/koji/taskinfo?taskID=221055

**Comment by lzap** [comment]
@orrabin we need a debian package as well, do you want to give it a try, or you want me to do it? :-D

File: rubygem-hammer_cli_foreman_discovery/foreman_discovery.yml [hunk 3]
```diff
@@ -0,0 +1,2 @@
+:foreman_discovery:
```
Comment (domcleal) [inline rubygem-hammer_cli_foreman_discovery/foreman_discovery.yml]: Please also submit this to the core repo, I'd like to be able to delete it in the next release.

**Comment by orrabin** [comment]
@lzap if you have time do it, it not I'll try to get to it later today

File: rubygem-hammer_cli_foreman_discovery/foreman_discovery.yml (hunk 3 above)
Comment (lzap) [inline rubygem-hammer_cli_foreman_discovery/foreman_discovery.yml]: You mean into the gem itself?

**Comment by domcleal** [comment]
Thanks @lzap, please just sort out the config file for the next release.

Is this for nightly, or 1.7 too?

**Comment by lzap** [comment]
Please 1.7 as well, will do.

File: rubygem-hammer_cli_foreman_discovery/foreman_discovery.yml (hunk 3 above)
Comment (domcleal) [inline rubygem-hammer_cli_foreman_discovery/foreman_discovery.yml]: Yep.  bootdisk is a bad example, but salt has it: https://github.com/theforeman/hammer_cli_foreman_salt

(Means we can re-use it between RPMs and debs, and the maintainer can add new options in the example file.)

**Pull request merged**